### DISCUSSION TITLE
Missing buttons "Create and run" and "Cancel" button

### DISCUSSION THREAD

**Discussion by bestoftheaverage** [TransformerOptimus/SuperAGI#316] [Q&A]
After installation everything seems to work in the GUI except no buttons available to Create and run or Cancel.
Suggestions on error and how to proceed?

**Comment by neelayan7** [comment]
Create and Run will be present when you create an Agent. 
![image](https://github.com/TransformerOptimus/SuperAGI/assets/43145646/459845b5-430a-4cc7-a4ef-546dbfa70f7e)
